Maestro/subscriptions.json

```diff
@@ -767,12 +767,9 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/ef6/blob/release/6.4//**/*",
-        "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
-        "https://github.com/dotnet/efcore/blob/release/5.0//**/*",
         "https://github.com/dotnet/efcore/blob/release/6.0//**/*",
         "https://github.com/dotnet/efcore/blob/release/7.0//**/*",
         "https://github.com/dotnet/razor-compiler/blob/release/6.0.2xx//**/*",
```

Comment:
I can't recall...is the trailing comma going to be an issue?

Reply:
Already removed it